src/Sulu/Component/Security/Authentication/UserInterface.php

```diff
@@ -18,6 +18,8 @@
  */
 interface UserInterface extends BaseUserInterface
 {
+    const RESOURCE_KEY = 'users';
```

Comment:
would be nice to adjust the existing places where this is hardcoded at the moment (`SecurityAdmin`, `UserController`, ...)